README.md

```diff
@@ -5,13 +5,28 @@ This repository is used for generalized Express.js discussions.
 * [Current discussions](https://github.com/expressjs/discussions/issues?q=is%3Aissue+is%3Aopen+label%3Adiscuss)
 * [Upcoming meetings](https://github.com/expressjs/discussions/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20label%3Ameeting)
 
-## Express.js TC meetings
+## Express.js meetings
 
 The Express.js Technical Committee (TC) meets online every other week, providing
 a live stream of the discussion and a recording of past meetings. Topics for the
 meetings are determined on an as-needed basis and may be technical or management
 of nature.
 
-Current meeting schedule: Wednesdays 23:30 UTC
+Current meeting schedule: 
+
+- TC Meeting: Monday 21:00 UTC (Every 2 weeks)
+- Working Session: Wednesday 21:00 UTC (Every 2 weeks)
```

Comment:
I wonder if we should just replace this with a link to the calendar or something? Either way, this seems fine to land as is now.